### DISCUSSION TITLE
wayland-hyprland-git exit status 4

### DISCUSSION THREAD

**Discussion by hasukomusicvpn** [prasanthrangan/hyprdots#79] [Q&A]
I'm install this rice and got a message: Failed to install the following packages. Manual intervention is required:
waybar-hyprland-git exxit status 4
I'm new to arch linux so I wonder if you can help me

**Comment by prasanthrangan** [comment]
Hi, can you try installing the non-git version of waybar -
```
yay -S waybar-hyprland
```

**Reply by hasukomusicvpn** [reply]
After installing waybar using that way, will the installation continue?

**Reply by prasanthrangan** [reply]
> After installing waybar using that way, will the installation continue?

please change it in `custom_hypr.lst` from `waybar-hyprland-git` to `waybar-hyprland` and rerun the install script.

**Reply by hasukomusicvpn** [reply]
Well i install the non-git version of waybar-hyperland, but same error![image](https://github.com/prasanthrangan/hyprdots/assets/90736680/8b9cdf61-a0b1-4206-931f-cd4c357d8ffb)

**Reply by prasanthrangan** [reply]
Hi, can you please try the steps suggested here - #82

**Reply by hasukomusicvpn** [reply]
Got another error, i think i installed the waybar  successfully ![image](https://github.com/prasanthrangan/hyprdots/assets/90736680/0d3fe316-cf3c-4df0-a2ea-2f4108c844aa)

**Reply by VimDiezel** [reply]
if you are installing a fresh minimal arch. After cloning the hyprdots, go to the scripts folder and edit the custom_apps.lst and add spdlog-git, then run the `./install.sh custom_apps.lst` that will do.

**Reply by hasukomusicvpn** [reply]
I got your rice installed, but something is wrong?![image](https://github.com/prasanthrangan/hyprdots/assets/90736680/e12be159-0c7a-451e-91cf-705d45f07bd1)

**Reply by VimDiezel** [reply]
Have you got it installed via this [method](https://github.com/prasanthrangan/hyprdots/discussions/79#discussioncomment-6767017) or you used another way.

**Reply by VimDiezel** [reply]
I guess this might be helpful #87.

**Reply by hasukomusicvpn** [reply]
How can i custom the wallpaper

**Reply by hasukomusicvpn** [reply]
> Have you got it installed via this [method](https://github.com/prasanthrangan/hyprdots/discussions/79#discussioncomment-6767017) or you used another way.

I got some error while installing  waybar, so i install the aur version, it worked well and i installed the rice using ./install.sh
